src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,143 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    private static class Node<T> {
+        private T value;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(Node<T> prev, T value, Node<T> next) {
+            this.prev = prev;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (size == 0) {
+            tail = new Node<>(null,value,null);
+            head = tail;
+        } else {
+            Node<T> newNode = new Node<>(tail,value,null);
+            tail.next = newNode;
+            tail = newNode;
+        }
```

Comment:
```suggestion
        Node<T> newNode = new Node<>(tail, value, null);
        if (size == 0) {
            head = newNode;
        } else {
            tail.next = newNode;
        }
        tail = newNode;
```
always try to move out all the code that repeats both in `if` and `else`